library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -559,7 +559,7 @@ public Iterator<Edge> edges(final Object... elementIds) {
             getOperation = new Builder()
                 .first(new GetAllElements.Builder()
                         .view(new View.Builder()
-                            .edges(graph.getSchema().getEdgeGroups())
+                            .allEdges(true)
```

Comment:
Might be worth changing the rest of the views in this file to the same as this for consistency?